plugin/contrib/mesos/pkg/scheduler/scheduler.go

```diff
@@ -154,29 +155,31 @@ type KubernetesScheduler struct {
 }
 
 type Config struct {
-	Schedcfg          schedcfg.Config
-	Executor          *mesos.ExecutorInfo
-	ScheduleFunc      PodScheduleFunc
-	Client            *client.Client
-	EtcdClient        tools.EtcdGetSet
-	FailoverTimeout   float64
-	ReconcileInterval int64
-	ReconcileCooldown time.Duration
+	Schedcfg             schedcfg.Config
+	Executor             *mesos.ExecutorInfo
+	ScheduleFunc         PodScheduleFunc
+	Client               *client.Client
+	EtcdClient           tools.EtcdGetSet
+	FailoverTimeout      float64
+	ReconcileInterval    int64
+	ReconcileCooldown    time.Duration
+	StaticPodsConfigPath string
```

Comment:
Why do we need this actually in the scheduler itself. As far as I see this is only used in the service.